app/editor/src/components/formik/select/FormikSelect.tsx

```diff
@@ -40,6 +39,7 @@ export const FormikSelect = <OptionType extends IOptionItem>({
         error={error}
         {...rest}
       />
+      <p role="alert">{error}</p>
```

Comment:
This was moved to the base component.

Reply:
It's seems although the Formik error messages conflict with the logic implemented at the base level. Should we keep these errors separate? To me I think it makes sense to have the Formik/Yup errors logic and display controlled in this component. If not, I can try to rewrite the base level so it doesn't override

Reply:
the display of the error should propagate to the base component.  All it should take is setting the base component with the error `error={error}` I think.